torchaudio/backend/utils.py

```diff
@@ -49,6 +49,10 @@ def set_audio_backend(backend: Optional[str]):
     if backend is None:
         module = no_backend
     elif backend == 'sox':
+        warnings.warn(
+            '"sox" backend is deprecated. '
```

Comment:
nit: I'd say "is being deprecated", though both seems fine :)

Reply:
changed it to `being deprecated`